config/400-config-istio.yaml

```diff
@@ -56,10 +83,44 @@ data:
     # sidecar injection (like Knative's istio-ci-no-mesh.yaml).  Since every pod
     # is outside of the service mesh in that case, a cluster-local  service
     # will need to be exposed to a cluster-local gateway to be accessible.
+    #
+    ##############
+    # NEW FORMAT #
+    ##############
+    # The configuration format should be
+    # ```
+    # local-gateways: |
+    #   - name: {{local_gateway_name}}
+    #     namespace: {{local_gateway_namespace}}
+    #     service: {{cluster_local_gateway_name}}.{{cluster_local_gateway_namespace}}.svc.cluster.local
+    # ```
+    # name, namespace & service are mandatory and can't be empty.
+    # Only one local gateway can be specified.
+    local-gateways: |
+      - name: knative-local-gateway
+        namespace: knative-serving
+        service: knative-local-gateway.istio-system.svc.cluster.local
+    #
+    ##############
+    # OLD FORMAT #
+    ##############
     # The configuration format should be `local-gateway.{{local_gateway_namespace}}.
     # {{local_gateway_name}}: "{{cluster_local_gateway_name}}.
     # {{cluster_local_gateway_namespace}}.svc.cluster.local"`. The
     # {{local_gateway_namespace}} is optional; when it is omitted, the system
     # will search for the local gateway in the serving system namespace
     # `knative-serving`
     local-gateway.knative-serving.knative-local-gateway: "knative-local-gateway.istio-system.svc.cluster.local"
+
+
+    # If no values are specified for the external traffic, it will default to:
```

Comment:
I'd probably keep the blocks together. Now this keys are kind of at the "old format" title.